data/sql/updates/pending_db_world/rev_1601314686326215700.sql

```diff
@@ -40,7 +40,7 @@ DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 11613);
 INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
 (11613, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 12, 11614, 1, 30000, 1, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - On Aggro - Summon Creature \'Bloodshot\''),
 (11613, 0, 1, 0, 0, 0, 100, 0, 2700, 3900, 9700, 12400, 11, 11976, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - In Combat - Cast \'11976\''),
-(11613, 0, 2, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 14443, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - Between 20-80% Health - Cast \'14443\' (No Repeat)');
+(11613, 0, 2, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 14443, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Huntsman Radley - Between 20-80% Health - Cast \'14443\' (No Repeat)');
```

Comment:
@Knindzagxg This one is still not working...  :,c

Just melees and her 11976..
![image](https://user-images.githubusercontent.com/61223313/94999867-a279b900-0579-11eb-9b9d-d55ff10142da.png)

Reply:
It's working, just make sure to make small macro and keep doing damage while away more then 5 yards xP

Reply:
@Knindzagxg Believe me, is not working..  :'c

![image](https://user-images.githubusercontent.com/61223313/95020697-e2987480-0629-11eb-9817-bbad6e309b31.png)